cluster/kube-up.sh

```diff
@@ -78,7 +78,24 @@ if [[ "${ENABLE_PROXY:-}" == "true" ]]; then
   echo ""
 fi
 
-echo -e "Done, listing cluster services:\n" >&2
+if [[ "${APISERVERS_EXTRA_NUM:-0}" -gt "0" ]]; then
```

Comment:
Is API Server partitioning optional for this setup script?  Looks like it's mandatory here when we have multiple API Servers. I think we should make it optional even there are multiple API Servers.

Reply:
it is optional, only when APISERVERS_EXTRA_NUM >0, that means we have multi apiservers.